ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/GSM.hs

```diff
@@ -420,7 +424,11 @@ realGsmEntryPoints tracerArgs gsmView =
     check . not =<< isHaaSatisfied
 
 data TraceGsmEvent selection
-  = -- | how many peers and the current selection
+  = -- | The GSM was initialized in the 'CaughtUp' state.
+    GsmEventInitializedInCaughtUp
+  | -- | The GSM was initialized in the 'PreSyncing' state.
+    GsmEventInitializedInPreSyncing
+  | -- | how many peers and the current selection
```

Comment:
Do we want to capitalize this and the following comment?

Reply:
Done